views/sis/balances.js

```diff
@@ -92,6 +97,13 @@ export default class BalancesView extends React.Component {
         let {flex, ole, print} = data
         this.setState({flex, ole, print})
       }
+      data = await getWeeklyMealsRemaining()
```

Comment:
Maybe don't reuse the variable? No harm, I suppose, but I dislike reusing variables in the same scope…